internal/pkg/server/server.go

```diff
@@ -40,6 +76,8 @@ func NewServer(opt ServerOpts) *Server {
 	})
 	g.POST("/tar", s.postTar)
 	g.GET("/tar", s.getTar)
+	g.POST("/tar/helm/repo", s.postHelmRepo)
+	g.POST("/tar/helm/chart", s.postHelmChart)
```

Comment:
I'd rather have `/helm/...` urls here. Especially the repo add does not make sense under `/tar` for me.